release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -86,6 +86,13 @@ You can now use the {product-title} Custom Metrics Autoscaler Operator to dynami
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-autoscaling-ingress-controller[Autoscaling an Ingress Controller].
 
+[id="ocp-4-12-nw-configure-dns-management"]
+==== Configuration of an Ingress Controller for manual DNS management
+
+You can now configure an Ingress Controller to stop automatic DNS management and start manual DNS management. Set `dnsManagementPolicy` to specify when it should be automatically or manually managed.
```

Comment:
> ! should verb
Whenever possible, rewrite in a more direct way. Do not use "should" to refer to an action that must be performed.

> Generally, when you specify the name of a command, parameter, or option in text, specify the word command, parameter, or option after the name.
Command, parameter, and option names are not translated; without the identifying noun, users might be confused about what the name represents.

Reply:
@kalexand-rh The main feature/doc PR has been approved by QE , the same addition to the rel notes, is QE ack required?